servers/docker-compose.local.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.9"
```

Comment:
For testing locally with 

```
docker compose -f docker-compose.yml -f docker-compose.local.yml
```

The template `docker-compose.yml` will get whatever fields are written in `docker-compose.local.yml` overwritten. Basically using this to inject environment variables as figuring out how to do environment variables with an .env file or AWS Secrets Manager seemed like it would take too long

Reply:
Testing locally with docker-compose is pretty useful since you can see all of the logs for all of the services in one terminal in pretty colors